src/neuroconv/datainterfaces/ecephys/kilosort/kilosortdatainterface.py

```diff
@@ -1,16 +1,14 @@
 """Authors: Heberto Mayorquin, Cody Baker."""
 from typing import Optional
 
-from spikeinterface.extractors import KiloSortSortingExtractor
-
 from ..basesortingextractorinterface import BaseSortingExtractorInterface
 from ....utils import FolderPathType
 
 
 class KilosortSortingInterface(BaseSortingExtractorInterface):
     """Primary data interface class for converting a KiloSortingExtractor from spikeinterface."""
 
-    SX = KiloSortSortingExtractor
+    ExtractorName = "KiloSortSortingExtractor"
```

Comment:
Do we need to define this because there is a difference in camel case with capital vs. lower case S? Perhaps it would be better to change the name of the class to be consistent with spikeinterface.

Reply:
Yes, that's why it's needed - we can definitely consider renaming to match SpikeInterface. Maybe in a follow-up PR though since this is enough as-is.